.flake8

```diff
@@ -7,4 +7,4 @@ select = C,E,F,W,B,B9
 # E203: whitespace before ':' (black behaviour)
 # E501: flake8 line length (covered by bugbear B950)
 # W503: line break before binary operator (black behaviour)
-ignore = E203,E501,W503
+ignore = E203,E501,W503,F402
```

Comment:
Why we are ignoring this? which module casuing this error?

Reply:
yes Removed the ignore F402
before
`path = _
for _ in xrange(7):`

after
`path = _
for path in xrange(7):`